sw.js

```diff
@@ -59,7 +42,7 @@ const externalRoute = new routing.RegExpRoute({
  * offline-cookbook/#stale-while-revalidate
  */
 const cdnAssetRoute = new routing.RegExpRoute({
-  regExp: new RegExp('^https:\/\/.*\.cloudfront\.net\/.*\.(png|svg)$'),
+  regExp: new RegExp('^https://.*.cloudfront.net/.*.(png|svg)$'),
```

Comment:
It looks like we might be up against a double-backslash situation again. 🙈 

I think this should be:
```js
new RegExp('^https://.*\\.cloudfront\\.net/.*\\.(png|svg)$')
```